editor/editor_plugin_settings.cpp

```diff
@@ -62,44 +63,33 @@ void EditorPluginSettings::update_plugins() {
 	plugins.sort();
 
 	for (int i = 0; i < plugins.size(); i++) {
-		Ref<ConfigFile> cf;
-		cf.instantiate();
+		Ref<ConfigFile> cfg;
+		cfg.instantiate();
 		const String &path = plugins[i];
 
-		Error err2 = cf->load(path);
+		Error err = cfg->load(path);
 
-		if (err2 != OK) {
-			WARN_PRINT("Can't load plugin config: " + path);
+		if (err != OK) {
+			WARN_PRINT("Can't load plugin config at " + path);
 		} else {
-			bool key_missing = false;
-
-			if (!cf->has_section_key("plugin", "name")) {
-				WARN_PRINT("Plugin config misses \"plugin/name\" key: " + path);
-				key_missing = true;
-			}
-			if (!cf->has_section_key("plugin", "author")) {
-				WARN_PRINT("Plugin config misses \"plugin/author\" key: " + path);
-				key_missing = true;
-			}
-			if (!cf->has_section_key("plugin", "version")) {
-				WARN_PRINT("Plugin config misses \"plugin/version\" key: " + path);
-				key_missing = true;
-			}
-			if (!cf->has_section_key("plugin", "description")) {
-				WARN_PRINT("Plugin config misses \"plugin/description\" key: " + path);
-				key_missing = true;
-			}
-			if (!cf->has_section_key("plugin", "script")) {
-				WARN_PRINT("Plugin config misses \"plugin/script\" key: " + path);
-				key_missing = true;
+			Vector<String> missing_keys;
+			for (const String required_key : { "name", "author", "version", "description", "script" }) {
+				if (!cfg->has_section_key("plugin", required_key)) {
+					missing_keys.append("\"plugin/" + required_key + "\"");
+				}
 			}
 
-			if (!key_missing) {
-				String name = cf->get_value("plugin", "name");
-				String author = cf->get_value("plugin", "author");
-				String version = cf->get_value("plugin", "version");
-				String description = cf->get_value("plugin", "description");
-				String scr = cf->get_value("plugin", "script");
+			if (missing_keys.size() > 0) {
+				String joined_keys = String(",").join(missing_keys);
```

Comment:
```suggestion
				const String joined_keys = String(",").join(missing_keys);
```
Though there is no need for a separate variable I think.

Reply:
A previous iteration of the code probably used this String more. Coding with explicit names is my game. But it is redudant now.